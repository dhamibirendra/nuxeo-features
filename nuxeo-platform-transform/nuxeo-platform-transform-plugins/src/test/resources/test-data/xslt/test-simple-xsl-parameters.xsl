<?xml version="1.0" encoding="UTF-8"?>
<xsl:stylesheet version="1.0"
  xmlns:xsl="http://www.w3.org/1999/XSL/Transform">

  <xsl:param name="param1" />
  <xsl:param name="param2" />
  <xsl:param name="param3" />

  <xsl:output method="xml"
      doctype-public="-//W3C//DTD XHTML 1.0 Transitional//EN"
     doctype-system="http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"/>
  <!--****************************************************************
      ** "/" template
      *************************************************************-->
  <xsl:template match="/">
    <html xmlns="http://www.w3.org/1999/xhtml">
      <head>
        <title>Test Parameters</title>
      </head>
      <body>
<h2>Hello</h2>
<table>
  <tbody>
    <tr>
      <td>Parameter 1:</td>
      <td><xsl:value-of select="$param1" /></td>
    </tr>
    <tr>
      <td>Parameter 2:</td>
      <td><xsl:value-of select="$param2" /></td>
    </tr>
    <tr>
      <td>Parameter 3:</td>
      <td><xsl:value-of select="$param3" /></td>
    </tr>
  </tbody>
</table>
</body>
</html>
</xsl:template>

</xsl:stylesheet>
