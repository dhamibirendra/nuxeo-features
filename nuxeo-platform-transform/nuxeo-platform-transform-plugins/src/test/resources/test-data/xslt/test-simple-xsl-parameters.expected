<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"><head><title>Test Parameters</title></head><body><h2>Hello</h2><table><tbody><tr><td>Parameter 1:</td><td>PARAM1</td></tr><tr><td>Parameter 2:</td><td>PARAM2</td></tr><tr><td>Parameter 3:</td><td>PARAM3</td></tr></tbody></table></body></html>